package.json

```diff
@@ -3836,7 +3836,7 @@
     "tsconfig-paths": "^3.11.0",
     "tslint": "^6.1.3",
     "typescript": "^4.1.5",
-    "vscode-cmake-tools": "^1.2.0",
+    "vscode-cmake-tools": "file:../vscode-cmake-tools-api",
```

Comment:
Builds/tests will fail until the https://github.com/microsoft/vscode-cmake-tools-api/pull/10 PR merges and this is updated to the new version

Reply:
Correct. This PR won't be merged until we merge the api, and update the npm package.